test/dash/dash_parser_manifest_unit.js

```diff
@@ -136,13 +136,15 @@ describe('DashParser.Manifest', function() {
                 .presentationTimeOffset(0)
                 .mime('video/mp4', 'avc1.4d401f')
                 .bandwidth(100)
+                .frameRate(23.80952380952380952380)
```

Comment:
The precision here is worrying.  Are all of these digits required to get a match out of Jasmine?